src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
@oliviertassinari are you sure it's okay to use this function with the muiTheme argument? I think it may throw errors when it's called from the `context-pure` since no theme is passed then.